src/defaults.scss

```diff
@@ -0,0 +1,6 @@
+
+.cal-event {
```

Comment:
Rather than having `defaults.scss` I think it's better to just have the CSS copied under each of the views, it also prevents adding the global `cal-event` rule to the whole app

Reply:
Also for the month view the background color is the primary color, and for the day / week views its the secondary color, and the border color is the primary one

Reply:
I understand. But wouldn't it be easier then, to override the cal-event once? Just to be sure, for me it doesn't matter but imo it makes it less "complicated"